tests/integration/cases/oracle-macaron/test.yaml

```diff
@@ -1,21 +1,22 @@
-# Copyright (c) 2024 - 2025, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2025 - 2025, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
 description: |
-  Analyzing with PURL and repository path without dependency resolution.
+  Analyzing the staging branch of the Macaron repo to detect vulnerable GitHub Actions.
```

Comment:
This test should be updated to use branch `main`.

Reply:
See commit [a684508](https://github.com/oracle/macaron/pull/1021/commits/a6845083b70a170af22416b1f8cce556e16ca5dc).